src/input.rs

```diff
@@ -147,8 +147,47 @@ supported_formats!(
     (toml, "toml-io", Toml [toml]) =>
         toml::from_str,
         toml::to_string,
+    (csv, "csv-io", Csv [csv]) =>
+        csv_from_str,
+        csv_to_string,
 );
 
+#[cfg(feature = "csv-io")]
+pub fn csv_from_str(_data: & str) -> Result<LanguageMap, Box<Error>>{
+    // not sure about the purpose of this...
```

Comment:
The purpose of this function is to provide user's with the ability to input previous results in the form of csv.